in_toto/gpg/common.py

```diff
@@ -0,0 +1,151 @@
+"""
+<Module Name>
+  gpg/common.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
```

Comment:
Is it a good idea to explain somewhere, maybe in the docstrings of the GPG modules, why other libraries can't be used for some of the functionality implemented here?  Is some of this not available in 'cryptography', 'gnupg,' etc.?   Is it to be compatible with Debian packaging?